R/scale_ggsurvfit.R

```diff
@@ -34,8 +34,23 @@
 #' # specify additional scales
 #' ggsurvfit +
 #'   scale_ggsurvfit(x_scales = list(breaks = seq(0, 8, by = 2)))
+#'
+#' @details
+#' Specical case: in the risk table, large numbers (with more than 4 digits) may not
+#' be shown completely, with some digits truncated outside the plot region.
+#' To remedy this, consider the following example code for a simulated large-size survival dataset:
+#' Here, adjust the expand size in `scale_ggsurvfit(x_scales = list(expand = c(0.05, 0)))` (for example)
+#' can modify the position of numbers in the risk table
+#' and make them all fit in the plot region. The scale of the `expand` argument differs by cases.
+#'
+#' df_colon_large <- df_colon[sample(1:nrow(df_colon), size = 15*nrow(df_colon), replace = TRUE), ]
+#' ggsurvfit <-
+#'   survfit2(Surv(time, status) ~ surg, data = df_colon_large) %>%
+#'   ggsurvfit() + add_risktable() +
+#'   scale_ggsurvfit()
```

Comment:
This looks great, thank you!

upon looking at it, perhaps we don't need the full example, and just keep this part: `scale_ggsurvfit(x_scales = list(expand = c(0.05, 0)))`